recipes/cfanalytics/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cfanalytics" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "aa3d98ebc1c39000c1e7ab08d586e079f8d7c87db15a318e03ff4918bc38c739" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Please use pip https://conda-forge.org/docs/meta.html#use-pip

Reply:
pip 10.0 is going to break conda with it's build isolation. We need to add a flag to pip to disable that.

Reply:
Thank you for the heads up.

Would you mind raising that issue elsewhere ([the io page](https://github.com/conda-forge/conda-forge.github.io) or [staged-recipes](https://github.com/conda-forge/staged-recipes))?
This issue seems a bit out of scope for this PR, and would most likely get more attention from the core developers and maintainers in those repos.

Reply:
I've cc'd you on pypa/pip#5033. I believe I understand exactly what is about to happen when pip 10.0 comes out, so feel free to ask questions there.